SETUP.md

```diff
@@ -388,17 +388,28 @@ Wondering where to start?  See our [contribution guidelines](CONTRIBUTING.md) fo
 ---
 ### Bundle Install Tips
 
-#### Apple Silicon (M1) bundle install steps
+#### Apple Silicon (M1) or Intel Mac bundle install steps
 
-On Apple Silicon, additional steps are required to get `bundle install` to work.
+On Apple Silicon/Intel Mac, additional steps are required to get `bundle install` to work.
 
-In Gemfile.lock, replace the two occurrences of libv8 (8.4.255.0) with libv8-node (15.14.0.0). Also update mini_racer to 0.4.0 (from 0.3.1):
+Simply run (if you're having issues only with part of gems in the command - you can run it with just needed gems)
+```
+bundle update sqlite3 libv8 mini_racer
```

Comment:
Can we update the sqlite gem instead (#52742 ) or remove it entirely (#52747 ) instead of documenting a workaround?

Reply:
yeah, that would be awesome. Don't know who's the best person to ask though :( @cat5inthecradle (?) @code-dot-org/platform (?)
Also, can we also update libv8 maybe or is there any reason we need specifically `libv8 (8.4.255.0)`

Reply:
For me, when updating `mini_racer` ends up updating `libv8`. 
And I think @fisher-alice ran into problems with sqlite when she removed that gem.
What has been fixing any issue for me was updating `mini_racer` and `sqlite3`

Reply:
FWIW, I don't think removing `sqlite3` was the issue for me. In the past, I used Rosetta mode for my setup, but could not seem to upgrade Ruby in Rosetta so installed v3.0.5 using arm. After deleting `sqlite3`, I was able to successfully run `bundle install` but not `bundle exec rake install:hooks`.

Reply:
We cannot update the versions of mini_racer and libv8 we install until we update our servers to a more-recent version

Reply:
Gotcha!
Should I update pr so it'll add `bundle update libv8 mini_racer` and merge it then?

Reply:
@Hamms aren't we able to update our servers to use libv8-node at this point since we did the node upgrade?

Reply:
Unfortunately, no; node is not the blocker here, updating to Ubuntu 20 is. Specifically, the version of the g++ compiler distributed with Ubuntu 18 incompatible with that library